docs/docker-integration/index.rst

```diff
@@ -0,0 +1,92 @@
+.. _docker-integration:
+
+======================
+Docker, Swarm, Compose
```

Comment:
This isn't a descriptive title. Perhaps "Installing Flocker with Docker Swarm and Docker Compose"